scripts/dockerhost/start-dockerhosts

```diff
@@ -1,5 +1,56 @@
 #!/bin/bash
 
+# check if all prequisites are installed before proceeding with setup.
+# Pre-requisites:
+# - Ubuntu 14+
+# - Docker 1.6+
+# - nsenter
+# - bridge utils
+# - ovs
+
```

Comment:
@mapuri - I see, this is not for the docker images themselves; clearly we don't need to run etcd in the host.